.ci/opensearch/Dockerfile.opensearch

```diff
@@ -5,10 +5,4 @@ ARG opensearch_path=/usr/share/opensearch
 ARG opensearch_yml=$opensearch_path/config/opensearch.yml
 
 ARG SECURE_INTEGRATION
-
```

Comment:
why remove these?

Reply:
This was added to this repo to solve a flaky spec issue myself when I first started. No other client repo has this. From what I've observed so far, it doesn't really help. If it happens again, I'm better equiped to find a better solution. This also adds complexity to the CI workflows esp now that HEALTHCHECK has to be aware of the OS version.